client/python/openlineage/client/transport/composite.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018-2024 contributors to the OpenLineage project
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import logging
+from functools import cached_property
+from typing import TYPE_CHECKING, Any
+
+import attr
+from openlineage.client.transport.transport import Config, Transport
+from openlineage.client.utils import get_only_specified_fields
+
+if TYPE_CHECKING:
+    from openlineage.client.client import Event
+
+log = logging.getLogger(__name__)
+
+
+@attr.s
+class CompositeConfig(Config):
+    transports: dict[str, dict[str, Any]] = attr.ib()
+    continue_on_failure: bool = attr.ib(default=True)
```

Comment:
Shouldn't we set this to False by default, so that the user has to explicitly allow failures to pass silently ?

Reply:
I would say it's better than other transports emit successfully even if some fail. Having OL event delivered is imho more desired than getting info in logs / throwing exception.

Reply:
I understand your point and partially agree - sending OL events should be the the most desired action. I'm still not convinced that we should do this by default, but maybe mentioning that behaviour in the docs is enough, so that the users are aware of it and can adjust this on their own. Feel free to resolve this one if that's okay.

Reply:
Added this to docstring. Also I checked how it renders to generated API reference:
![image](https://github.com/user-attachments/assets/23419264-97f9-4875-8511-5a6d601c6f5c)